README.md

```diff
@@ -1 +1,66 @@
-# Awesome-Books-With-ES6
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Description the project.
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+
+Demo - https://divyanshthinks.github.io/Awesome-Books-With-ES6/?#
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```

Comment:
- [OPTIONAL] It will be nice to remove all sections of your readme file that is unrelated to your project as this will make your readme file clean and professional. I mean sections on which you actually have information at the moment, but update the sections if information is available. Also remember to update your project title.